modules/controlfreec/1.1/config/default.yaml

```diff
@@ -3,12 +3,12 @@ lcr-modules:
     controlfreec:
 
         inputs:
-            # Available wildcards: {seq_type} {genome_build} {sample_id}
+            # Available wildcards: {seq_type} {genome_build} {tumour_id} {normal_id}
             sample_bam: "__UPDATE__"
             sample_bai: "__UPDATE__"
 
 
-        scratch_subdirectories: []
+        scratch_subdirectories: ["mpileup"]
```

Comment:
same here, I think it should be `scratch/mpileup`

Reply:
This should only be set as a scratch directory if the demo is setup to use scratch... is it?

Reply:
Yes, my current demo repo looks like this:

scratch -> /projects/rmorin_scratch/jawong_tmp/

results/controlfreec-1.1
├── 00-inputs
│   └── genome--hg38
├── 01-mpileup -> ../../scratch/controlfreec-1.1/01-mpileup
├── 02-run
│   └── genome--hg38
├── 99-outputs
│   └── genome--hg38
└── logs
    ├── launched-2020-11-05-at-00-06-02
    ├── launched-2020-11-05-at-09-59-48
    ├── launched-2020-11-05-at-10-00-11
    ├── launched-2020-11-05-at-10-38-58
    ├── launched-2020-11-05-at-10-39-52
    ├── launched-2020-11-05-at-10-40-00
    ├── launched-2020-11-05-at-10-43-04
    ├── launched-2020-11-05-at-10-46-34
    ├── launched-2020-11-05-at-10-47-43
    ├── launched-2020-11-05-at-10-51-01
    └── launched-2020-11-05-at-10-51-39

Reply:
The scratch 01-mpileup symlink was automatically generated when doing my test run